content/news/vcmi-1.2.0-release.md

```diff
@@ -0,0 +1,42 @@
+description = ""
+featured = "vcmirelease1_1.jpg"
+featuredalt = ""
+featuredpath = "date"
+linktitle = ""
+title = "VCMI 1.2.0 released"
+type = "post"
+forumurl = "https://forum.vcmi.eu/t/vcmi-1-1-0-released/5843"
+
++++
+
+## Biggest update yet!
+ 
+VCMI project is gaining momentum recently. Thanks to extraordinary activity of developers and supporting community we are able to present you the biggest release yet, containing tons of new features as well as bug fixes. This time we definitely focused on user experience, trying to polish all the features and rectify the issues encountered in the past.
+
+## Rendering improvements
+The rendering engine got a rewrite, resulting in smooth and correct animations, as well as overall better performance and lower CPU consumption. All the animations should have exactly same timing and order as in original game, and adventure map scrolls smoothly.
+
+## Revamped GUI
+We introduced a number of quality-of-like GUI improvements known from popular HD Mod, as well as custom features and options that will make the game more accessible and fun to play. All the features are customisable and can be disabled depending on your preference. There is new in-game menu with more options than ever before. The highlights include expanded RMG options window, optional display of adventure map rewards in Status Window as well as extra tooltips on adventure map and in the battle.
+
+## Translation support
+VCMI now supports translations - both with original game content and mods. Game will automatically pick preferred language configured in Launcher and enable translations of the content that allows it. Many existing mods from our repository have already been translated thanks to combined effort of the community.
+ 
+## New user-friendly Launcher
+Following numerous reports and complaints, our Launcher got complete overhaul. New players will now be guided through installation and configuration process, which should reduce confusion and help you get the game running quickly with the best configuration possible.
+
+## Bugfixes
+A lot of effort was spent on fixing and polishing the game as much as possible. Hundreds of issues were found and fixed, especially in battle visuals and mechanics. We did our best to ensure players' experience will not be disrupted by unexpected bugs. Of course you may also observe tweaks in Random Map Generator and AI, as well as about any aspect of the game.
+
+## Modding
+Mod system got a few subtle, but significant extensions. Capabilities of configurable adventure map objects were expanded, as well as secondary skills and related bonuses.
+
+**Please note that save games from v1.1 are incompatible with v1.2!**
+ 
+## And many others
+The list of changes is too big to fit here, please refer to the he detailed backlog is on [the release page](https://github.com/vcmi/vcmi/releases/tag/1.1.0). **TODO**: Update link
+
+But the journey doesn't end here, more features are already scheduled for 1.3 release.
```

Comment:
I think it should be "But the journey doesn't **ends** here". Or not?

We could also list some of that stuff we have in works - moats & quick combat replay are already in, hota maps are also very likely to be in

Reply:
it should be not due to usage of "doesn't"

i still suggest mentioning we got discord...

Reply:
Ah, I keep converting "doesn't" into "does not" mentally so got this result 😅 

Fully agree on discord link.

Reply:
Okay, will add this one.